frontend/apps/filemanager/filemanagerbookinfo.lua

```diff
@@ -400,6 +393,7 @@ function BookInfo:setCustomMetadata(file, book_props, metadata_updated_caller_ca
         book_props.display_title = nil
         custom_doc_settings:saveSetting("doc_props", book_props) -- save a copy of original props
     end
+    local prop_value_old = custom_props[prop_key]
```

Comment:
Will `prop_value_old` be the original metadata value if it was never customized before?
(I see `custom_props = {}` a few lines above.)

Reply:
My mistake.